app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -76,7 +75,8 @@ class ProfileEditFragmentPresenter @Inject constructor(
     profileEditViewModel.profile.observe(
       fragment,
       Observer {
-        activity.title = it.name
+        if (activity is ProfileEditActivity)
```

Comment:
Ditto here (for braces). Also, could we use isMultipane as a proxy for this, instead? That gives information about how the fragment is hosted.

Reply:
done